### ISSUE TITLE
The issue of accuracy after model fine-tuning.

### ISSUE THREAD

**Issue by lab-bj** [moojink/openvla-oft#58]
I performed model fine-tuning on 4 × 4090 GPUs using the following command:

```
torchrun --standalone --nnodes 1 --nproc-per-node 4 vla-scripts/finetune.py --vla_path openvla/openvla-7b --data_root_dir /newdataf/modified_libero_rlds/ --dataset_name libero_goal_no_noops --run_root_dir /newdataf/checkpoint/log/ --use_l1_regression True --use_diffusion False --use_film False --num_images_in_input 2 --use_proprio True --batch_size 1 --learning_rate 5e-4 --num_steps_before_decay 100000 --max_steps 50005 --save_freq 2000 --save_latest_checkpoint_only True --image_aug True --lora_rank 8 --run_id_note parallel_dec--8_acts_chunk--continuous_acts--L1_regression--3rd_person_img--wrist_img--proprio_statde --wandb_entity "1342607442" --wandb_project "vla"
```

The process took approximately 48 hours, but the accuracy of the fine-tuned model was only 27.20%.

**Comment by moojink** [comment]
Closing the issue due to inactivity, but please reopen if the issue persists!
